e2e/nsenter.go

```diff
@@ -84,47 +83,22 @@ func extractLogsFromVM(ctx context.Context, t *testing.T, cloud *azureClient, ku
 	for file, sourceCmd := range commandList {
 		mergedCmd := fmt.Sprintf("%s %s", sshCommand, sourceCmd)
 		cmd := append(nsenterCommandArray(), mergedCmd)
-
-		req := kube.typed.CoreV1().RESTClient().Post().Resource("pods").Name(podName).Namespace(defaultNamespace).SubResource("exec")
-
-		option := &corev1.PodExecOptions{
-			Command: cmd,
-			Stdout:  true,
-			Stderr:  true,
-		}
-
-		req.VersionedParams(
-			option,
-			scheme.ParameterCodec,
-		)
-
-		exec, err := remotecommand.NewSPDYExecutor(kube.rest, "POST", req.URL())
+		stdout, stderr, err := execOnPod(ctx, kube, defaultNamespace, podName, cmd)
 		if err != nil {
 			return nil, err
 		}
 
-		var stdout, stderr bytes.Buffer
-
-		err = exec.StreamWithContext(ctx, remotecommand.StreamOptions{
-			Stdout: &stdout,
-			Stderr: &stderr,
-		})
-
-		t.Log("stdout")
+		t.Log("----------------------------------- begin stdout -----------------------------------")
```

Comment:
should we even bother doing this? it was a stop gap mostly

Reply:
figured it would be nice to have the output directly in the pipeline logs, though can remove if we want to everything from the uploaded logs

Reply:
actually yeah I think it makes more sense to only output stderr here